lib/domain/models/omrs_patient.dart

```diff
@@ -17,7 +18,7 @@ class OmrsPatient {
 
   fhir.Patient toFhir() {
     return fhir.Patient(
-      id: fhir.Id(uuid),
+      fhirId: FhirId(uuid).toString(),
```

Comment:
also you don't need to case uuid to fhirId and back to string again ..
just fhirId: uuid should suffice